stable/postgresql/Chart.yaml

```diff
@@ -1,6 +1,6 @@
 apiVersion: v1
 name: postgresql
-version: 4.2.0
+version: 4.2.1
```

Comment:
I'm afraid another patch version was released and you'll need to bump the version to **4.2.2**

Reply:
Thanks for the review!

I just bumped the version to 4.2.2.